Animations/include/core.h

```diff
@@ -1,6 +1,11 @@
 #ifndef MATH_ANIM_CORE_H
 #define MATH_ANIM_CORE_H
 
+// _MAX_PATH is not a good constant, it lies, gets ignored sometimes, and is also not necessarily defined.
+#ifndef _MAX_PATH
+#define _MAX_PATH 4096
```

Comment:
Why specifically 520, anyway?

Reply:
PATH_MAX and friends has been deprecated since c99, since the limit can not always be known at compile time. It can change between kernel/OS build versions or even be dynamic. pathconf() can be used to fetch value in runtime

Reply:
I think `pathconf` is Linux specific, but I'll be trying to find a Windows alternative. 

And I chose 520 because in Windows `_MAX_PATH` is 260 which accounts for the total number of characters, not bytes. The characters could be UTF-16, so if `_MAX_PATH` was accurate, then you would need 520 bytes to hold all the data. It's really just arbitrary anyways until I come up with a robust solution